backend/uclapi/oauth/views.py

```diff
@@ -94,17 +94,21 @@ def shibcallback(request):
     try:
         # Expire our signed tokens after five minutes for added security
         appdata = signer.unsign(appdata_signed, max_age=300)
-    except signing.BadSignature:
+    except signing.SignatureExpired:
         response = PrettyJsonResponse({
             "ok": False,
-            "error": "Bad signature. Please try login again."
+            "error": ("Login data has expired. Please attempt to log in again."
+                      " If the issues persist please contact the UCL API"
```

Comment:
Much nicer! What does PEP 8 say about the spacing? Should we have
```
"...log in again. "
"If the issues..."
```
OR
```
"...log in again."
"If the issues..."
```

It's subtle, but we used different formats and I want us to be as close to PEP 8 as possible :P

Reply:
PEP 8 unfortunately doesn't comment on multi line strings and wether to use parenthesis or concatenation etc.. Its probably best to do the former though, more readable and its usually bad practice to start with whitespace. I'll update it